src/main/kotlin/sliv/tool/parsers/points_parsers/CSVPointsParser.kt

```diff
@@ -0,0 +1,35 @@
+package sliv.tool.parsers.points_parsers
+
+import com.fasterxml.jackson.dataformat.csv.CsvMapper
+import com.fasterxml.jackson.dataformat.csv.CsvSchema
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import sliv.tool.data_structures.landmarks.Point
+import sliv.tool.data_structures.landmarks.PointLandmark
+import sliv.tool.parsers.ParserUtils
+
+class CSVPointsParser : IPointsParser {
```

Comment:
looks like only mapping and scheme is different from lines - lets add some base class with common logic for them?